### ISSUE TITLE
Use serial_test

### ISSUE THREAD

**Issue by BaptisteGelez** [Plume-org/Plume#469]
[This crate](https://github.com/palfrey/serial_test) allows you to run some test one after the other. By using it we could make sure database tests are always run serially (without even needing to set `RUST_TEST_THREADS=1`), and other tests can still run concurrently.

Plume version (and/or commit): 0.2.0

**Comment by igalic** [comment]
why are our database tests not parallelizable?
fixing that issue would be a better solution than cheating on the database tests

**Comment by BaptisteGelez** [comment]
I agree, and I don't remember the details… :sweat_smile: maybe @fdb-hiroshima nows, since she is the one who wrote most of these tests.

**Comment by fdb-hiroshima** [comment]
I have no idea, running theme in parallel on my computer (or anything other than Travis really) works fine, but for some reasons, it did not work at all on Travis, one random test was passing, and all others failed

**Comment by BaptisteGelez** [comment]
Ah, they don't work concurrently on my computer either… Really weird :woman_shrugging:

**Comment by igalic** [comment]
so, when looking into deploying Plume, please consider doing it on @fdb-hiroshima's machine, but not on Travis (docker!) or on @BaptisteGelez's machine.

**Comment by fdb-hiroshima** [comment]
(Plume is using virtual machines from GCE, not docker containers when building on Travis)
